src/App.tsx

```diff
@@ -0,0 +1,19 @@
+// @ts-ignore
+import Routes from 'components/Routes.tsx';
+// @ts-ignore
+import React from 'react';
+// @ts-ignore
```

Comment:
ts-ignore should be used for some exceptional cases only, seems like the app not fully configured for using typescript